### DISCUSSION TITLE
Repository secrets; does it have to be plaintext 'stringData' or can it be encoded 'data'?

### DISCUSSION THREAD

**Discussion by kencieszykowski** [argoproj/argo-cd#7606] [Q&A]
Hey all!

We're currently running Argo v2.0.0 in EKS utilizing [Kubernetes External Secrets](https://github.com/external-secrets/kubernetes-external-secrets) to interface with Amazon Secrets Manager.  

We're looking to upgrade to v2.1.6; [which moves Repos and Repo Creds into secrets.](https://argo-cd.readthedocs.io/en/stable/operator-manual/upgrading/2.0-2.1/#replacing-repositories-and-repositorycredentials-with-secrets)

The problem here is that in the [examples](https://argo-cd.readthedocs.io/en/stable/operator-manual/argocd-repo-creds.yaml) that the page links to, it explicitly looks for `stringData` (plaintext) instead of the standard encoded `data` field.  Is this just an oversight in documentation, or is that the only way to get a working secret?  Nearly every other secret related to Argo uses encoded 'data' values, and this is really throwing me off.

It seems like the person in [this post](https://github.com/external-secrets/kubernetes-external-secrets/issues/821) is attempting to do the same thing.

Any help would be much appreciated!

**Comment by reggie-k** [comment]
Using data in the repo secret works for me
